### PR TITLE
add method to retrieve app org parameters

### PR DESCRIPTION
**Pull request by GuiMartins**

### CODE REVIEW

**Review by jaikme — APPROVED** [review]

File: packages/org/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# `@mamba/org`
+
+O pacote `@mamba/org` contém métodos que aplicam as customizações espeficicas para cada organização.
```
Comment (FPTeixeira) [inline packages/org/README.md:3]:
```suggestion
O pacote `@mamba/org` contém métodos que aplicam as customizações específicas para cada organização.
```

File: packages/org/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Image
+
+O componente `Image` trata as imagens espeficicas de cada organização através da tag `img`.
```
Comment (FPTeixeira) [inline packages/org/README.md:7]:
```suggestion
O componente `Image` trata as imagens específicas de cada organização através da tag `img`.
```

File: packages/org/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| class        | Classe CSS                 | `string`   | ``          |
+
+## ORG
+
+`import * as ORG from '@mamba/org/org.js';`
+
+### `getAppOrgParams(defaultAppOrgParams: string)`
+
+`ORG.getAppOrgParams()` faz o merge do arquivo json default do app com o arquivo fornecido pelo backend, substituindo o `value` de cada `key` se forem do mesmo tipo. Análogo ao `lodash.merge`.
+
```
Comment (FPTeixeira) [inline packages/org/README.md:21]:
```suggestion
`ORG.getAppOrgParams()` faz o merge do arquivo json default do app com o arquivo fornecido pelo backend, substituindo o `value` de cada `key` se forem do mesmo tipo. Análogo ao `lodash.merge`, porém levemente diferente. Nessa implementação o merge substitui objetos do tipo Array, enquanto o lodash os mescla.

Caso o arquivo de configuração do backend não esteja acessível (arquivo não encontrado no POS ou execução no desktop), o arquivo default será usado.

```

File: packages/org/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+## Image
+
+O componente `Image` trata as imagens específicas de cada organização através da tag `img`.
+
+| Propriedades | Descrição                  | Tipo       | Padrão      |
+| ------------ | -------------------------- | ---------- | ----------- |
+| fileName     | Nome da imagem             | `string`   | `null`      |
+| class        | Classe CSS                 | `string`   | ``          |
+
+## ORG
```
Comment (FPTeixeira) [inline packages/org/README.md:14]:
```suggestion
O componente `Image` trata as imagens específicas de cada organização através da tag `img`.

| Propriedades | Descrição                  | Tipo       | Padrão      |
| ------------ | -------------------------- | ---------- | ----------- |
| fileName     | Nome da imagem             | `string`   | `null`      |
| class        | Classe CSS                 | `string`   | ``          |

Caso ocorra um erro no carregamento da imagem (onerror) (ex: arquivo não encontrado no POS ou a execução está acontecendo no desktop), a propriedade `src` da tag `img` será redirecionada automaticamente para o caminho da imagem padrão (`/src/assets/org/{fileName}`).

## ORG
```

**Review by FPTeixeira — COMMENTED** [review]
Revisado!

File: packages/org/components/Image.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<img class="{className}" src="{path}/{fileName}" onerror="this.src='{defImgPath}/{fileName}';" />
```
Comment (GuiMartins) [inline packages/org/components/Image.html:1]:
```suggestion
<img class="{className}" src="{path}/{fileName}" onerror="this.src='{defImgPath}/{fileName}';" alt="" />
```

**Pull request merged**